### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by unconst**
1) Removes the parser generation on the config with messes with --help when using a parser
2) Turns off console-rich local logging (which sucks)

### CODE REVIEW

**Review by eduardogr — APPROVED** [review]
LGTM

File: bittensor/_config/__init__.py [hunk 1]
```diff
@@ -54,7 +54,7 @@ def __new__( cls, parser: ArgumentParser = None, strict: bool = False, args: Opt
                 Nested config object created from parser arguments.
         """
         if parser == None:
-            parser = ArgumentParser()
+            return config_impl.Config()
```
Comment (camfairchild) [inline bittensor/_config/__init__.py:57]: Why is this changed?

Comment (unconst) [reply bittensor/_config/__init__.py:57]: Because it messes with --help when importing bittensor and using argparse anywhere else.

Comment (unconst) [reply bittensor/_config/__init__.py:57]: This is because we create an argument parser on bittensor import
